meta/meta.go

```diff
@@ -427,6 +430,19 @@ func (m *Meta) GetSchemaVersionWithNonEmptyDiff() (int64, error) {
 	return v, err
 }
 
+// Returns the timestamp of a schema version, which is the commit timestamp of the schema diff
+func (m *Meta) GetTimestampForSchemaVersion(helper *helper.Helper, version int64) (int64, error) {
+	diffKey := m.schemaDiffKey(version)
+	data, err := helper.GetMvccByEncodedKey(m.txn.EncodeStringDataKey(diffKey))
```

Comment:
I don't recommend to insert the logic here, maybe in `domain.go`

Reply:
Move this function to `domain.go` and avoid import `github.com/pingcap/tidb/store/helper` in this library.